assets/js/components/ClusterDetails/ClusterSettings.test.jsx

```diff
@@ -42,4 +42,84 @@ describe('ClusterDetails ClusterSettings component', () => {
       screen.getByRole('button', { name: 'Select Checks for Execution' })
     ).toBeVisible();
   });
+
+  it('given VMware provider, should render the warning banner', async () => {
+    const group = faker.animal.cat();
+    const catalog = catalogCheckFactory.buildList(2, { group });
+
+    const [StatefulChecksSettings, state] = withState(<ClusterSettings />, {
+      ...defaultInitialState,
+      catalog: { loading: false, data: catalog, error: null },
+      clustersList: {
+        clusters: defaultInitialState.clustersList.clusters.map((cluster) => ({
+          ...cluster,
+          provider: 'vmware',
+        })),
+      },
+    });
+
+    const {
+      clusters: [, , , { id: clusterID }],
+    } = state.getState().clustersList;
+
+    render(
```

Comment:
Can we try to replaces this `render` by our internal `renderWithRouter` and set the route there?
I don't know if it will work, but at least let's try

Reply:
Anyway, forget about this by now. I found out that it is not that trivial